src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const result = {};
```

Comment:
The variable name 'result' is not descriptive of its purpose. Consider using a more descriptive name like 'styleObject' to improve code readability.